libraries/abstractions/common_io/test/test_iot_i2c.c

```diff
@@ -484,7 +487,7 @@ TEST( TEST_IOT_I2C, AFQP_IotI2CReadAsyncAssisted )
         TEST_ASSERT_EQUAL( IOT_I2C_SUCCESS, lRetVal );
 
         /* Read from i2c device */
-        lRetVal = iot_i2c_read_async( xI2CHandle, &ucReadBuf, sizeof( ucReadBuf ) );
+        lRetVal = iot_i2c_read_async( xI2CHandle, ucReadBuf, sizeof( ucReadBuf ) );
```

Comment:
Were these tests passing earlier?

Reply:
They weren't running. See the test case name is suffixed with "assisted". We haven't support assisted tests yet.

Reply:
Is it possible to locally verify that these tests are passing?

Reply:
Yes, but it takes extra work. Since assisted test is not supported for now, let's worry about this later.

Reply:
Since ucReadBuf is an array this would work either way. I agree with Qiutong, this should not hold us up here. The warning this removes is simply that the adress off operator is redundant.